src/app/api/deps.py

```diff
@@ -55,9 +55,8 @@ async def get_current_access(security_scopes: SecurityScopes, token: str = Depen
     if entry is None:
         raise unauthorized_exception("Invalid credentials", authenticate_value)
 
-    for scope in security_scopes.scopes:
-        if scope not in token_data.scopes:
-            raise unauthorized_exception("Permission denied", authenticate_value)
+    if set(token_data.scopes).isdisjoint(security_scopes.scopes):
```

Comment:
Perhaps a more readable solution: 
```
if not any(scope in token_data.scopes for scope in security_scopes.scopes):
```

Reply:
Question of perspective :-)  However, from a python user point of view, your suggestion might be more readable !
Let's wait another reviewer point of view but honestly I do not have any preference

Reply:
No no, really, if both implementations are equivalent behaviour-wise, let's go with yours!